viewer/packages/data-providers/src/image-360-data-providers/Cdf360ImageAnnotationProvider.ts

```diff
@@ -94,28 +96,47 @@ export class Cdf360ImageAnnotationProvider implements Image360AnnotationProvider
     });
   }
 
-  private async getFilesByAssetRef(assetRef: IdEither): Promise<CogniteInternalId[]> {
+  private async getFilesByAssetRef(assetRef: IdEither | DMInstanceRef): Promise<CogniteInternalId[]> {
+    const annotationData = this.getAnnotationDataByAssetRefType(assetRef);
+
     const response = await this._client.annotations
       .reverseLookup({
         limit: 1000,
         filter: {
           annotatedResourceType: 'file',
-          annotationType: 'images.AssetLink',
-          data: {
-            assetRef
-          }
+          annotationType: annotationData.type,
+          data: annotationData.data
         }
       })
       .autoPagingToArray();
 
     return response.map((a: AnnotationsAssetRef) => a.id).filter((id): id is number => id !== undefined);
   }
 
+  private getAnnotationDataByAssetRefType(assetRef: IdEither | DMInstanceRef): {
+    type: 'images.InstanceLink' | 'images.AssetLink';
+    data: {
+      instanceRef?: DMInstanceRef;
+      assetRef?: IdEither;
+    };
+  } {
```

Comment:
[Nitpick]: I would maybe want `data` to be `{ instanceRef: DMInstanceRef } | { assetRef: IdEither }` instead